### PR TITLE
Memorize install directory on Linux

### PR DESCRIPTION
**Pull request by oatmealine**
Port of [powercord-org/powercord#602](https://github.com/powercord-org/powercord/pull/602)'s first half (as the second half seems to have already been implemented, separately):

> Currently, if a user is using a non-standard Discord install location, they will have to enter it each time they inject and uninject Powercord. This PR will fix this by creating an `.installdir` file and using that if possible.

Seperate `.installdir` files will be made for different Discord branches - for instance, Canary will have `.installdir-canary` and PTB will have `.installdir-ptb`

### CODE REVIEW

File: injectors/linux.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-const { existsSync } = require('fs');
+const { existsSync, readFileSync, writeFileSync } = require('fs');
 const { execSync } = require('child_process');
 const readline = require('readline');
 const { BasicMessages, AnsiEscapes, PlatformNames } = require('./log');
 
 // This is to ensure the homedir we get is the actual user's homedir instead of root's homedir
 const homedir = execSync('grep $(logname) /etc/passwd | cut -d ":" -f6').toString().trim();
 
+const installDirFile = __dirname + '/../.installdir-';
```
Comment (asportnoy) [inline injectors/linux.js:10]: path.join preferred for this

**Review by asportnoy — COMMENTED** [review]
LGTM for the most part

File: injectors/linux.js (hunk 1 above)
Comment (oatmealine) [reply injectors/linux.js]: Should be good as of 86e558b

**Review by lleyton — APPROVED** [review]
LGTM, nya

**Pull request merged**